inc/commonitiltask.class.php

```diff
@@ -229,6 +229,11 @@ function post_deleteFromDB() {
 
    function prepareInputForUpdate($input) {
 
+      if (empty($input['content'])) {
+         Session::addMessageAfterRedirect(__("You can't add a task without description"),
+                                          false, ERROR);
+         return false;
+      }
```

Comment:
Maybe this check should be done only if `content` is defined in input. Indeed, it should be possible to update only specific fields without having to pass any `content` field in `$input`.

Reply:
I fixed this.

Reply:
In update, content is always in the input even you doesn't change it, so your test in not useful.
But your message in better